lib/lighthouse/benefits_documents/service.rb

```diff
@@ -86,6 +92,13 @@ def submit_document(file, file_params, lighthouse_client_id = nil) # rubocop:dis
         )
       end
 
+      if Flipper.enabled?(:benefits_documents_validate_claimant) && !validate_claimant_can_upload(document_data)
```

Comment:
Does this `validate_claimant_can_upload(document_data)` return a faraday response instead of a boolean?